src/content/docs/en/hardware/anywhere.mdx

```diff
@@ -22,14 +24,16 @@ By contributing to Ultramarine Anywhere, you can help us bring Linux to more dev
 
 - [Android Devices](/en/anywhere/android)
 - [Apple Silicon Macs](/en/anywhere/applesilicon)
+- [RISC-V](/en/anywhere/riscv)
+- [Steam Deck](/en/anywhere/steamdeck)
+- [Data Centre, Enterprise, and HPC](/en/anywhere/enterprise)
```

Comment:
once again, keep these in wishlist, we don't have the hardware and this is a ton of work

moving them to planning makes it seem like support is coming faster than it is